modules/hal_nordic/nrfs/backends/Kconfig

```diff
@@ -25,4 +25,19 @@ config NRFS_BACKEND_TX_MSG_QUEUE_SIZE
 	range 1 16
 	default 8
 
+config NRFS_SEND_RETRY_DELAY
+	int "Delay in us"
+	range 1 100
+	default 1
```

Comment:
calling `k_usleep(1)` does not make much sense since currently systick is 31250Hz when GRTC is used so it's hard to say how long will it take and cpu will definitely won't sleep during that time. If 1-100us range is reasonable maybe it's better to use `k_busy_wait` there?

Reply:
This is more like give small delay and let other tasks run simultaneously (this smallest delay in most cases is enough for sysctrl to handle incoming messages). We have test where we send a lot of messages without delay and then ipc service may return ENOMEM, adding even this k_usleep(1) will unlock ipc queue.